packages/terra-alert/src/Alert.scss

```diff
@@ -0,0 +1,149 @@
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+.terra-Alert {
```

Comment:
Specifically you need to [wrap the selectors with :local](https://github.com/cerner/terra-core/blob/master/packages/terra-badge/src/Badge.scss#L4]) and in the [JSX file import the styles object](https://github.com/cerner/terra-core/blob/master/packages/terra-badge/src/Badge.jsx#L5) and reference class names from that as properties.